packages/cli/package.json

```diff
@@ -13,11 +13,15 @@
     "dist"
   ],
   "dependencies": {
-    "@prisma/sdk": "2.24.1",
     "@redwoodjs/api-server": "^0.33.2",
     "@redwoodjs/internal": "^0.33.2",
     "@redwoodjs/prerender": "^0.33.2",
     "@redwoodjs/structure": "^0.33.2",
+    "@redwoodjs/dev-server": "^0.33.0",
+    "@redwoodjs/eslint-config": "^0.33.0",
+    "@redwoodjs/core": "^0.33.0",
```

Comment:
```suggestion
    "@redwoodjs/dev-server": "^0.33.2",
    "@redwoodjs/eslint-config": "^0.33.2",
    "@redwoodjs/core": "^0.33.2",
```

Reply:
Also, I just noticed Redwood packages regained the `^`, which I assume is from Lerna.

Feels like we should pin as well. (If so do you know if/how to manage that for Lerna?) Either way will defer to your judgement.

Reply:
Good point - I'll look into lerna's docs to figure this out!